pyproject.toml

```diff
@@ -167,6 +182,7 @@ ignore = [
     "S106",    # good idea but too many false positives on non-passwords
     "S107",    # good idea but too many false positives on non-passwords
     "S603",    # not going to manually mark every subprocess call as reviewed
+    "S606",    # yes, I mean to start a subprocess without a shell
```

Comment:
This is a weirdly misnamed diagnostic in Ruff, but I think what it's actually complaining about is not using subprocess. This is generally a good diagnostic for all of our other code but not for this code since you want to use execlp (which I suspect is what it's firing on). My convention for things like that is to move the suppression down into the path-specific suppressions and suppress it for `src/lsst/rsp/*`. That makes it clear that this is an exception for this specific package, and makes it easier to reconcile the suppression list with other packages.